api/src/main/resources/liquibase.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                  http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+    <changeSet author="slubwama" id="patientqueue_1566208965241-1">
+        <createTable tableName="patient_queue">
+            <column autoIncrement="true" name="patient_queue_id" type="INT">
+                <constraints primaryKey="true" unique="true"/>
+            </column>
+            <column name="patient_id" type="INT"/>
+            <column name="provider_id" type="INT"/>
+            <column name="location_from" type="INT"/>
+            <column name="location_to" type="INT"/>
+            <column name="encounter_id" type="INT"/>
+            <column name="status" type="VARCHAR(255)"/>
+            <column name="date_created" type="datetime"/>
+            <column name="changed_by" type="INT"/>
+            <column name="date_changed" type="datetime"/>
+            <column name="voided" type="TINYINT(3)"/>
```

Comment:
What does it mean to void a patient queue?

Reply:
It means that that queue was deleted